backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/internal/BpkDialogImpl.kt

```diff
@@ -202,6 +218,29 @@ private fun DialogIcon(icon: Dialog.Icon?) {
     }
 }
 
+private fun Modifier.dialogWidth(): Modifier =
+    widthIn(
+        max = DialogMaxWidth,
+    ).padding(
+        top = IconPadding,
+        start = BpkDimension.Spacing.Lg,
+        end = BpkDimension.Spacing.Lg,
+    )
+
+private fun Modifier.contentSize(size: Dp, setSize: (Dp) -> Unit): Modifier =
```

Comment:
I think the whole modifier can be simplified a lot - i don't think we'll need any external parameters here (see next comment)